matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
```

Comment:
I still need wbm_updateState in the linearization code, but in balancing I think you can remove both if "normal mode" is used.